utilities/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openvswitch/ovn:2.12_e60f2f2_debian_master
+
+ARG GOVERPKG=go1.13.9.linux-amd64.tar.gz
+RUN apt-get update
+RUN apt-get install -y wget git gcc python3-pip python3-dev build-essential autoconf automake libtool
```

Comment:
what's the purpose of installing the tools such as gcc, python, build-essential, etc.?

Reply:
> Thanks @vjayaramrh
> It seems the #121 is addressing the problem in a similar way. Could you take a look?
> Please also see my comments.

@hzhou8 appreciate you pointing this out, I have reached out to the PR author to have an initial discussion and look at ways for combining our efforts

Reply:
The test.sh script in the `.travis` folder needs them when executed inside the container

Reply:
but according to the steps you put in README, the test.sh script is not used. Your approach is supposed to run "go test" directly.

Reply:
The README captures the native `go` way of running tests. Do you think that the README should also capture the alternate way i.e executing the `.travis/test.sh`? If you think that would be useful to capture as well, I can certainly update the README